Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -169,6 +169,8 @@ class TopSitesPanel: UIViewController {
     }
 
     private func refreshHistory(frequencyLimit: Int) {
+        // We double the requested limit in order to generate some leeway for "grouping by domain". Hopefully
+        // that returns enough entries that we can group but still have frequencyLimit results.
```

Comment:
Oops. This comment was left over from earlier, but I do need to update this for the different frecency model we talked about.